src/bridge/cockpitfsread.c

```diff
@@ -357,6 +370,8 @@ cockpit_fsread_open (CockpitTransport *transport,
   options = json_object_new ();
   json_object_set_string_member (options, "path", path);
   json_object_set_string_member (options, "payload", "fsread1");
+  if (g_str_equal (binary, "raw"))
```

Comment:
This is just for testing, right?  I'd actually expect you to set this if `binary` was non-`NULL`.